packages/run-protocol/src/vaultFactory/liquidateMinimum.js

```diff
@@ -55,14 +57,20 @@ const start = async zcf => {
       // check whether swapIn liquidated assets until getOfferResult() returns.
       // Of course, we also can't exit the seat until one or the other
       // liquidation takes place.
-      await deposited;
-      await E(liqSeat).getOfferResult();
+      const [offerResult, amounts, _] = await Promise.all([
+        E(liqSeat).getOfferResult(),
+        E(liqSeat).getCurrentAllocation(),
+        deposited,
+      ]);
+      trace('offerResult', { offerResult, amounts });
 
       // if swapOut failed to make the trade, we'll sell it all
       const sellAllIfUnsold = async () => {
-        if (
-          !AmountMath.isEqual(inBefore, debtorSeat.getAmountAllocated('In'))
-        ) {
+        const unsold = debtorSeat.getAmountAllocated('In');
+        // TODO XXX This could be true if it gave me more collateral. This
```

Comment:
There's no way currently to make that happen. It was more an issue when we were considering incrementally adding more in-progress liquidation assets.

However it's worth noting that this scenario exists because I cannot get the "satisfied/rejected" state from making an offer. We need that API (@erights). That's especially opaque in the AMM and with `offerTo`

Reply:
I changed the comment.